README.md

```diff
@@ -25,22 +23,21 @@ NA
 1. Clone the repository to your local machine:
 
 ```bash
-git clone https://github.com/bikashamdocs/Ansible_Repo_Final.git
+git clone https://github.com/bikashamdocs/ansible-setup.git
 ```
-2. Change directory to the cloned repository:
 
+2. Modify the inventory file prod_host.ini or dev_host.ini to specify the target host or group of hosts to install required software.
 
-3. Modify the inventory file prod_host.ini or dev_host.ini to specify the target host or group of hosts to install Nginx on.
 
-4. Run the playbook to install Nginx:
+3. Run the playbook to install Nginx and kubernetes:
 
 ```bash
-ansible-playbook -i inventory/prod_host.ini ./my-playbook/site.yml --extra-vars "@group_vars/<your-environment>/all.yml"
+ansible-playbook -i /path/to/my-playbook/inventory/<your-environment>_host.ini /path/to/my-playbook/nginx.yml --extra-vars "/path/to/my-playbook/@group_vars/<your-environment>/all.yml"
```

Comment:
To enhance user convenience, instead of using absolute paths in Ansible playbooks, relative paths can be used. Also, the location of the inventory file and any extra variables file can be specified in the ansible.cfg file. This allows the user to specify these locations once and have Ansible use them for all playbooks, eliminating the need to specify them in each playbook.